system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -24,9 +24,6 @@
 namespace system_metrics_collector
 {
 
-/* static */ constexpr const std::chrono::milliseconds PeriodicMeasurementNode::
-INVALID_PUBLISH_WINDOW;
```

Comment:
revert

Reply:
This has been moved to `metrics_message_publisher.cpp`.